protocol/avdl/stellar1/local.avdl

```diff
@@ -336,6 +336,57 @@ protocol local {
     boolean pending;
   }
 
+  // Check but do not send a request.
+  // Arguments are optional unless marked [required] or otherwise described.
+  // However, a coherent payment is required in order for res.readyToSend to returned as true.
+  BuildRequestResLocal buildRequestLocal(
+    int sessionID,
+
+    // Recipient username, acctID, or assertion.
+    // Examples: "", "russel", "GBQTE2V7Y356TFBZL6YZ2PA3KIILNSAAQRV5C7MVWS22KQTS4EMK7I46"
+    string to,
+
+    string amount, // Amount of the asset OR outside currency being requested. Example: "3.005"
+    // Zero or one of `asset` and `currency` should be set.
+    // If both are null or `asset` is native then XLM are being requested.
+    // If `currency` is set then XLM is being requested after converting `amount`.
+    // Note: Sending non-XLM assets is not yet supported.
```

Comment:
s/Sending/Requesting
It may actually be supported, but that would be a weird asymmetry, so whatever you want.